tests/steamship_tests/utils/test_generation_utils.py

```diff
@@ -160,7 +161,8 @@ def create_test_game_state(context: AgentContext) -> GameState:
 
 
 def run_onboarding(context: AgentContext):
-    agent = OnboardingAgent(tools=[])
+    secrets = toml.load("../../src/.steamship/secrets.toml")
```

Comment:
question: does this path need to be specified relative the file itself?

Reply:
It works for me in the context of a PyCharm run configuration, which defaults to the directory marked as test root.  If other people are doing this differently, I'm happy to convert.